server/src/services/s3.js

```diff
@@ -0,0 +1,54 @@
+import AWS from "aws-sdk";
+
```

Comment:
When we're creating `services/`, it's probably best practice to instantiate a [singleton](https://refactoring.guru/design-patterns/singleton). You probably don't need multiple instances of `s3`.